src/handlers/swap_handlers.ts

```diff
@@ -123,5 +129,6 @@ const parseGetSwapQuoteRequestParams = (req: express.Request): GetSwapQuoteReque
     const buyAmount = req.query.buyAmount === undefined ? undefined : new BigNumber(req.query.buyAmount);
     const gasPrice = req.query.gasPrice === undefined ? undefined : new BigNumber(req.query.gasPrice);
     const slippagePercentage = Number.parseFloat(req.query.slippagePercentage || DEFAULT_QUOTE_SLIPPAGE_PERCENTAGE);
-    return { takerAddress, sellToken, buyToken, sellAmount, buyAmount, slippagePercentage, gasPrice };
+    const discluded = req.query.discluded === undefined ? undefined : parseStringArrForERC20BridgeSources(req.query.discluded.split(','));
+    return { takerAddress, sellToken, buyToken, sellAmount, buyAmount, slippagePercentage, gasPrice, discluded };
```

Comment:
nit: discluded isn't a word :P. `excludedSources`?

Reply:
Wait discluded isn't a word? I poached that from dex.ag and looked it up; ill use excludedSources lol

Reply:
Ok it is a word haha, but I've never seen it used. Interesting that dex.ag uses it.